server/routes/induction/common/inPrisonTrainingController.ts

```diff
@@ -18,19 +28,10 @@ export default abstract class InPrisonTrainingController extends InductionContro
 
     this.addCurrentPageToFlowHistoryWhenComingFromCheckYourAnswers(req)
 
-    if (req.session.pageFlowHistory) {
```

Comment:
The "pageFlowHistory" is part of the complex implementation of how the Induction journey knows what the previous page is.
Again, when all Induction pages have been migrated to the JS back link solution this is another thing I'm looking forward to nuking completely 😁